### ISSUE TITLE
Greets.  I'm getting an error?

### ISSUE THREAD

**Issue by spiraloid** [knekke/blender_addons#1]
minor bullet proofing note:
If I have the renderer set to blender don't have a world lightmap set, I get an error at line 191.

**Comment by knekke** [comment]
What file and what is the error?
I guess you are talking about the obj importer. It doesn't work for Blender internal, because there is no principled shader. But I will see if I can print a nice warning message instead of letting it crash when I do an update

**Comment by spiraloid** [comment]
Yeah, I figured out it was my own user error eventually.   Thanks for the
response!

I modified your code a little since the Obj's i'm bringing in from 3Dcoat
were pretty huge and rotated in the wrong coordinate frame for me.  odd
that changing the axis_forward didn't fix it.


load_obj(obj, objects, materials, objname)
obj_objects = bpy.context.selected_objects[:]
for obj in obj_objects:
bpy.ops.object.select_all(action='DESELECT')
obj.scale = (0.01,0.01,0.01)
obj.select = True
bpy.context.scene.objects.active = obj
bpy.ops.object.transform_apply(location=False, rotation=True, scale=True)
return {'FINISHED'}


feel free it use if you like.

cheers!

-bay
